default/generated/spring-framework/spring-framework-5.x-to-6.0-web-applications.yaml

```diff
@@ -49,3 +49,50 @@
     url: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-6.0-Release-Notes#web-applications
   - title: 'Specific Spring Framwork change'
     url: https://github.com/spring-projects/spring-framework/issues/28552
+
+- ruleID: spring-framework-5.x-to-6.0-web-applications-00030
+  category: mandatory
+  effort: 1
+  labels:
+    - konveyor.io/source=spring5
+    - konveyor.io/target=spring6+
+  when:
```

Comment:
We should add another condition to tackle consuming a REST endpoint that returns a Source object via a RestTemplate. For that, we need to look for the usage of the getForObject method from the RestTemplate class with a Class<javax.xml.transform.Source.class> parameter.

Reply:
This won't work at the moment due to https://github.com/konveyor/analyzer-lsp/issues/736